go/service/login.go

```diff
@@ -28,8 +30,21 @@ func (h *LoginHandler) GetConfiguredAccounts(context context.Context, sessionID
 	return h.G().GetConfiguredAccounts(context)
 }
 
-func (h *LoginHandler) Logout(ctx context.Context, sessionID int) (err error) {
+func (h *LoginHandler) Logout(ctx context.Context, arg keybase1.LogoutArg) (err error) {
 	defer h.G().CTraceTimed(ctx, "Logout [service RPC]", func() error { return err })()
+
+	if !arg.ForceDangerous {
+		res, err := libkb.CanLogout(libkb.NewMetaContext(ctx, h.G()))
+		if err != nil {
+			return err
+		}
+		if !res.CanLogout {
+			return fmt.Errorf("cannot logout: %s; try with force", res.Reason)
```

Comment:
I'm not sure about this error message especially if it ends up in GUI.

Even in cli, I don't think encouraging users to just `--force`  it is a good idea.